@xen-orchestra/web/src/components/host/HostHeader.vue

```diff
@@ -0,0 +1,36 @@
+<template>
+  <HeadBar>
+    {{ host.name_label }}
+    <template #icon>
+      <ObjectIcon type="host" :state="host.power_state.toLocaleLowerCase() as HostState" />
+    </template>
+  </HeadBar>
```

Comment:
I think it would be preferable to define the content in order of appearance.

```suggestion
  <HeadBar>
    <template #icon>
      <ObjectIcon type="host" :state="host.power_state.toLocaleLowerCase() as HostState" />
    </template>
    {{ host.name_label }}
  </HeadBar>
```